packages/salesforcedx-vscode-core/src/conflict/persistentStorageService.ts

```diff
@@ -63,6 +66,15 @@ export class PersistentStorageService {
     }
   }
 
+  public setPropertiesForFilesDeploy(result: DeployResult) {
```

Comment:
mostly a nit, but we pass a DeployResponse here and a FileProperties in the set for the retrieve.  Seems like it's the same data, just one layer deeper on the retrieve method.  Did you consider if we could just make `setPropertiesForFilesRetrieve` -> `setPropertiesForFiles` and reuse that same logic?

Reply:
The data is coming from a different place though - the fileProperty for retreive (fileProperty.lastModifiedDate) and from the response itself for the deploy (result.response.lastModifiedDate).  I kinda think these should stay the way they are to keep it clear that the internal logic is actually slightly different for each operation.

Reply:
word that makes sense.  I was assuming a DeployResponse and a RetrieveResponse would have the same shape but I guess not.